services/AuthService.js

```diff
@@ -59,13 +60,18 @@ module.exports = class AuthService {
     const confirmationToken = jwt.sign({
       email: email
     }, process.env.JWT_EMAIL_SECRET, {
-      expiresIn:  60 * 60,
+      expiresIn: 60 * 60,
       algorithm: 'HS256'
-    });
+    })
 
-    // send email verification  
-    const url = `http://localhost:8080/verify-email/${confirmationToken}`;
-    mailService.sendMail(email, url);
+    // send email verification
+    const url = `http://localhost:8080/verify-email/${confirmationToken}`
```

Comment:
In this case, "http://localhost:8080" shuld be dynamic, for example u can save that url in env file (proccess.env.SITE_URL)